### ISSUE TITLE
Meal Planner - Remove image and have standard sort order in calendar view

### ISSUE THREAD

**Issue by bartekd123** [TandoorRecipes/recipes#1272]
When I look at the calendar, I have 2 issues, wondering if you can incorporate these changes

1.) I have 3 categories (Breakfast, lunch and dinner). When I add Lunch first, then breakfast, they show up in the order they were added. Can it instead be shown in the calendar view consistently throughout?

2.) To keep it concise, can there be an option to not show an image for each meal in the calendar view?

**Describe the solution you'd like**
1.) Change the order based on the order configured in the settings
2.) Option to not display images

**Comment by vabene1111** [comment]
1. this should already be the case but we will verify this. 
2. some sort of "compact layout" even for the large view could be configurable

we will look into this

**Comment by m-oezokyay** [comment]
(1) still happens to me in the compact layout when viewing the meal plan on my smartphone (version 1.5.4). I think we should label this as a bug, if it isn't already fixed in a development version.

**Comment by manjarco** [comment]
Having the same problem with meal type order (on the hosted instance). 
Hoped changing the mealtype order a few times in the settings was a solution. Sadly its not.

![calendar-view](https://github.com/TandoorRecipes/recipes/assets/274524/85642015-5b3e-4a8c-8c42-0bed01f4ba6c)

![list-view](https://github.com/TandoorRecipes/recipes/assets/274524/46637e29-ac2a-460f-b106-35c6a54b8a9d)

**Comment by vabene1111** [comment]
yes there is currently no sorting between entry types. I added the ability for a "time" with the next version which is not yet represented in the UI but a preparation for future calendar export. I will look into sorting and times with the new interface.

**Comment by tmllull** [comment]
Hi! Is this issue in the roadmap? On the "normal" web, the order on the meal plan is correct (following the configured order), but on mobile the order seems that follows the "added time" (old to new):

Correct order (red is breakfast, green is "second breakfast")
![Image](https://github.com/user-attachments/assets/be140671-b81d-4c38-aeb4-eca05f709d39)

Incorrect order
![Image](https://github.com/user-attachments/assets/58fd29e8-a5ef-4491-a44a-fe0e75b5e25b)

I'm with self-hosted version
